keyboards/tronguylabs/m122_3270/config.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2021 James R. Maynard III <jaymaynard@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID       0x1209
```

Comment:
pid.codes vendor id 👍🏻

Reply:
It's on request there, but they haven't merged the PR yet...if they turn it down, I'll pick something else.

Reply:
Ive added the `awaiting PR` label while this is sill unresolved.

Reply:
For quick reference: https://github.com/pidcodes/pidcodes.github.com/pull/656